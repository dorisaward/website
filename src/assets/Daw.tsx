import { SVGProps } from 'react'

export const Daw = ({
    width,
    height,
    stroke = 'black',
}: SVGProps<SVGSVGElement>) =>
    <svg width={width} height={height} viewBox="0 0  1080 590" preserveAspectRatio="xMinYMin meet" >
        <rect id="svgEditorBackground" x="0" y="0" width="1080" height="590" style={{ fill: 'none', stroke: 'none' }} />
        <path d="M80.52,177.66a256.27,256.27,0,1,1,7.01,251.37" style={{ fill: 'none', stroke, strokeWidth: 0 }} id="e3_circleArc" />
        <path d="M89.53,166.65a258.01,258.01,0,1,1,-6.01,259.38" style={{ fill: 'none', stroke, strokeWidth: 10 }} id="e4_circleArc" />
        <path d="M90.53,163.64a130.84,130.84,0,1,1,-5.01,261.39" style={{ fill: 'none', stroke, strokeWidth: 10 }} id="e9_circleArc" />
        <path d="M221.724362,163.64a152.088526,152.088526,0,0,1,-5.01,261.39" style={{ fill: 'none', stroke, strokeWidth: 10 }} id="e10_circleArc" />
        <path d="M218.719911,160.635549a130.886917,130.886917,0,1,1,-5.01,261.39" style={{ fill: 'none', stroke, strokeWidth: 10 }} id="e11_circleArc" />
        <path d="M477.111484,490.125935a170.77,170.77,0,0,1,-3,-174.26" style={{ fill: 'none', stroke, strokeWidth: 10 }} id="e12_circleArc" />
        <path d="M474.107032,491.127419a163.454509,163.454509,0,0,0,-3,-174.26" style={{ fill: 'none', stroke, strokeWidth: 10 }} id="e13_circleArc" />
        <circle id="e13_circle" cx="475.104" cy="202.703" style={{ fill: 'none', stroke, strokeWidth: 10 }} r="22.17" />
        <polyline style={{ stroke, fill: 'none', strokeWidth: 10 }} id="e15_polyline" points="550.22,60.487 605.298,227.743 648.363,53.4748 700.436,234.752 729.478,52.4781" />
        <polyline style={{ stroke, fill: 'none', strokeWidth: 10 }} id="e16_polyline" points="616.31,553.219,660.375,356.927,711.454,547.209" />
        <path d="M554.22,55.48a255.14,255.14,0,0,0,63.09,495.73" style={{ fill: 'none', stroke, strokeWidth: 10 }} id="e17_circleArc" />
        <path d="M730.49,57.49a255.94,255.94,0,0,1,-18.93,485.72" style={{ fill: 'none', stroke, strokeWidth: 10 }} id="e18_circleArc" />
    </svg>