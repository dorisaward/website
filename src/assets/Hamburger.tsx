import { SVGProps } from 'react'

type HamburgerProps = {
    /**
     * If not defined, set to the height
     * If a string, strokeLinecap = 'round' is not visible to user and looks like strokeLinecap = 'butt'
     */
    width: SVGProps<SVGSVGElement>['width']
    height: number
    strokeWidth: number
    stroke?: SVGProps<SVGSVGElement>['stroke']
}

export const Hamburger = ({ width, height, strokeWidth, stroke = 'black' }: HamburgerProps) => {
    if (!width) {
        width = height
    }
    const x1 = typeof width === 'number' ? strokeWidth : 0
    const x2 = typeof width === 'number' ? width - strokeWidth : width
    const y1 = strokeWidth
    const y2 = height / 2
    const y3 = height - strokeWidth
    return (
        <svg width={width} height={height} viewBox={`0 0 ${width} ${height}`}>
            <line x1={x1} y1={y1} x2={x2} y2={y1} stroke={stroke} strokeWidth={strokeWidth} strokeLinecap={'round'}/>
            <line x1={x1} y1={y2} x2={x2} y2={y2} stroke={stroke} strokeWidth={strokeWidth} strokeLinecap={'round'}/>
            <line x1={x1} y1={y3} x2={x2} y2={y3} stroke={stroke} strokeWidth={strokeWidth} strokeLinecap={'round'}/>
        </svg>
    )
}