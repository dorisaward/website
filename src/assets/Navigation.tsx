import { Link } from 'react-router-dom'
import styled from 'styled-components'

const StyledNav = styled.nav`
    display: flex;
    flex-direction: column;
    justify-content: space-evenly;
    height: 70%;

    a:link { text-decoration: none; }
    a:visited { text-decoration: none; }
    a:hover { text-decoration: none; }
    a:active { text-decoration: none; }
`

const LinkContainer = styled.div`
    padding: ${({theme}) => theme.padding};
    margin: ${({theme}) => theme.padding};
    background-color: ${({theme}) => theme.tertiaryColour};
    border-radius: ${({theme}) => theme.padding};
    box-shadow: ${({theme}) => '1px 1px ' + theme.shadowColour};

    &:active { box-shadow: unset; }
`

export const Navigation = () => {
    return (
        <StyledNav>
            <LinkContainer><Link to="/">Home</Link></LinkContainer>
            <LinkContainer><Link to="/cv">Curriculum Vitae</Link></LinkContainer>
        </StyledNav>
    )
}
