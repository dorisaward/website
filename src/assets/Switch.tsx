import styled from 'styled-components'

const StyledLabel = styled.label<{ checked: boolean }>`  
    cursor: pointer;
    text-indent: -9999px;
    width: 50px;
    height: 25px;
    background: ${({ checked, theme }) => (checked ? theme.switchOnColour : theme.switchOffColour)};
    display: block;
    border-radius: 20px;
    position: relative;&:after {
        content: "";
        position: absolute;
        left: ${({ checked }) => (checked ? "7px" : "calc(55% - 3px)")};
        top: 3px;
        width: 20px;
        height: 20px;
        background: ${({ theme }) => theme.backgroundColour};
        border-radius: 18px;
        transition: 0.3s;
    }
`

const Container = styled.div`
    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: center;
`

const Text = styled.p`
    padding: ${({ theme }) => theme.padding}
`

type SwitchProps = {
    checked: boolean,
    onSwitch: () => void,
    label: { left: string, right: string } | string
}

export const Switch = ({ checked, onSwitch, label }: SwitchProps) => {
    const id = typeof label === 'string' ? label : (label.left + label.right)
    return (
        <Container key={'switch' + id}>
            {typeof label !== 'string' ? <Text>{label.left}</Text> : <Text>{label}</Text>}
            <StyledLabel htmlFor={id} checked={checked}>
                <input
                    id={id}
                    type="checkbox"
                    checked={checked}
                    onChange={onSwitch}
                />
            </StyledLabel>
            {typeof label !== 'string' && <Text>{label.right}</Text>}
        </Container>
    )
}
