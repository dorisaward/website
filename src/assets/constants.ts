export const MOBILE_WIDTH = '480px'