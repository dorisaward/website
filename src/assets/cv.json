{
  "data": [
    {
      "heading": "Developer",
      "text": "Full Stack Developer with 7 years experience writing React Native apps for ios and android and React apps for web.\nIndustry experience in the software development lifecycle (SDLC) in development, decision-making, leading, testing, deploying, bug fixing, and designing user experiences.\nProfessional writer of React Native, React, Typescript, JavaScript, Redux, Realm, HTML5, CSS, PHP; using tools such as Jira, Git, Bitbucket, Trello; and following methodologies such as Agile and SCRUM.\nExperience mentoring less-experienced devs, leading on smaller projects, maintaining legacy systems, decision making of code trajectory, reviewing code to ensure high quality, estimating effort/time required for tasks, supporting non-developer team members such as Product Owners, Scrum Masters, UX/UI Designers.\nExcellent analytical and logical programming skills with a good understanding at the conceptual level."
    },
    {
      "heading": "Work Experience",
      "text": "January 2024 - Current\nDeveloper \nMayden, Bath, UK\nReact.JS and PHP development of new features and information security fixes. Maintenance of legacy systems and implementation of new features. Mentoring devs. Leading on sub-projects. Decision making on trajectory of the frontend codebases.\n\nApril 2020 - December 2023\nDeveloper\nLivingWith, Bath, UK\nFixed bugs and added new features to react-native TS apps. Maintained a high quality code base by following coding standards and kept in mind the flexibility required for improvements in the future. Reviewed and tested code and added unit tests to reduce the likelihood of future bugs.\n\nJuly 2018 - April 2020 \nSoftware Engineer \nLivingMap, Bath, UK\nDeveloped front-end and UI across multiple map products across mobile and desktop platforms. Lead feature development of the Map Management System being delivered by multiple engineers. Development of applications using TypeScript, JavaScript, CSS, React.\n"
    },
    {
      "heading": "Contact",
      "text": "07432682616\ndorisaward@gmail.com\n3 Whitemore Court,\nBatheaston,\nBath, BA1 7PD\ndorisward.uk\nReferences available on request\n"
    },
    {
      "heading": "Education",
      "text": "2013 - 2014\nPGCE: Secondary Mathematics\nKeele University\n\n2010 - 2013\nBSc(Hons) Computer Science &\nBSc(Hons) Mathematics Dual Hons\nKeele University\n\n2008 - 2010\n5x A-Levels (B-C Grade)\nJohn Taylor Sixth Form \n\n2003 - 2008\n10x GSCEs(A* - A)\nJohn Taylor High School\n"
    }
  ]
}