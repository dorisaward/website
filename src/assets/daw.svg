<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="1080" height="1080" viewBox="0 0 1080 1080" xml:space="preserve">
<desc>Created with Fabric.js 5.2.4</desc>
<defs>
</defs>
<g transform="matrix(1 0 0 1 540 540)" id="5ddeee4a-3005-4305-a944-4f4f03b8a9f9"  >
</g>
<g transform="matrix(1 0 0 1 540 540)" id="85375e3c-465b-48e3-a3b5-f3e55cfff137"  >
<rect style="stroke: none; stroke-width: 1; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: rgb(255,255,255); fill-rule: nonzero; opacity: 1; visibility: hidden;" vector-effect="non-scaling-stroke"  x="-540" y="-540" rx="0" ry="0" width="1080" height="1080" />
</g>
<g transform="matrix(1.32 0 0 1.32 604.15 540)"  >
<g style="" vector-effect="non-scaling-stroke"   >
		<g transform="matrix(1 0 0 1 0 0)" id="svgEditorBackground"  >
<rect style="stroke: none; stroke-width: 1; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: none; fill-rule: nonzero; opacity: 1;" vector-effect="non-scaling-stroke"  x="-540" y="-295" rx="0" ry="0" width="1080" height="590" />
</g>
		<g transform="matrix(1 0 0 1 -217.98 2.18)"  >
<path style="stroke: rgb(0,0,0); stroke-width: 0; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: none; fill-rule: nonzero; opacity: 1;" vector-effect="non-scaling-stroke"  transform=" translate(-322.02, -297.18)" d="M 80.5 177.7 C 134.81874623228265 74.61919579090153 251.83225401301732 21.148988155560517 365.31867383999696 47.54995221932404 C 478.80509366697663 73.95091628308757 560.2026137760672 173.57856218425678 563.4456608095529 290.05027992972583 C 566.6887078430386 406.5219976751949 490.9610845937657 510.524647568045 379.11958054769866 543.1996588359111 C 267.27807650163163 575.8746701037771 147.47051754711504 528.9985255835786 87.50000000000028 429.1000000000005" stroke-linecap="round" />
</g>
		<g transform="matrix(1 0 0 1 -214.53 6.55)"  >
<path style="stroke: rgb(0,0,0); stroke-width: 10; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: none; fill-rule: nonzero; opacity: 1;" vector-effect="non-scaling-stroke"  transform=" translate(-325.47, -301.56)" d="M 89.5 166.7 C 150.4545014365763 67.28430994178417 270.5179237986608 21.115636216694327 382.3766889743568 54.07845357961574 C 494.2354541500528 87.04127094253715 570.0767519122398 190.9396317408548 567.3801458067029 307.52290237023965 C 564.683539701166 424.10617299962456 484.1195198209503 524.3868317707472 370.85631757678976 552.142510334443 C 257.5931153326292 579.8981888981389 139.79274328370792 528.2276540841283 83.5 426.09999999999997" stroke-linecap="round" />
</g>
		<g transform="matrix(1 0 0 1 -385.62 -0.61)"  >
<path style="stroke: rgb(0,0,0); stroke-width: 10; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: none; fill-rule: nonzero; opacity: 1;" vector-effect="non-scaling-stroke"  transform=" translate(-154.38, -294.39)" d="M 90.5 163.6 C 138.2893050996203 162.8837647709895 182.66437160422868 188.29023837052267 206.2400067825335 229.86571312123775 C 229.8156419608383 271.4411878719528 228.83775071489575 322.56534220983224 203.6894497162986 363.2088365440037 C 178.54114871770145 403.85233087817517 133.2269584294795 427.54324868595563 85.50000000000011 425" stroke-linecap="round" />
</g>
		<g transform="matrix(1 0 0 1 -284.87 -0.7)"  >
<path style="stroke: rgb(0,0,0); stroke-width: 10; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: none; fill-rule: nonzero; opacity: 1;" vector-effect="non-scaling-stroke"  transform=" translate(-255.13, -294.3)" d="M 221.7 163.6 C 267.264468529914 191.89497271545906 294.5582293437994 242.0965026140747 293.53249651011765 295.7218151589541 C 292.5067636764359 349.3471277038335 263.3131829941125 398.4681805271581 216.69999999999993 425" stroke-linecap="round" />
</g>
		<g transform="matrix(1 0 0 1 -256.21 -3.57)"  >
<path style="stroke: rgb(0,0,0); stroke-width: 10; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: none; fill-rule: nonzero; opacity: 1;" vector-effect="non-scaling-stroke"  transform=" translate(-283.79, -291.43)" d="M 218.7 160.6 C 267.0612638449584 159.01558788287443 312.3520546875092 184.238926305242 336.46878639844863 226.18776960301534 C 360.58551810938803 268.1366129007887 359.59409964051906 319.96797045326423 333.8908197142807 360.9638678513212 C 308.1875397880425 401.95976524937805 261.96529836613433 425.4326631158074 213.7000000000002 422" stroke-linecap="round" />
</g>
		<g transform="matrix(1 0 0 1 -75.62 107.95)"  >
<path style="stroke: rgb(0,0,0); stroke-width: 10; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: none; fill-rule: nonzero; opacity: 1;" vector-effect="non-scaling-stroke"  transform=" translate(-464.38, -402.95)" d="M 477.1 490.1 C 444.29362529331763 436.9224027787121 443.14306883181547 370.0750723654336 474.1 315.79999999999995" stroke-linecap="round" />
</g>
		<g transform="matrix(1 0 0 1 -55.55 108.95)"  >
<path style="stroke: rgb(0,0,0); stroke-width: 10; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: none; fill-rule: nonzero; opacity: 1;" vector-effect="non-scaling-stroke"  transform=" translate(-484.45, -403.95)" d="M 474.1 491.1 C 506.73992604685105 437.2677049458797 505.57313534054947 369.4771649097588 471.1 316.79999999999995" stroke-linecap="round" />
</g>
		<g transform="matrix(1 0 0 1 -64.9 -92.3)" id="e13_circle"  >
<circle style="stroke: rgb(0,0,0); stroke-width: 10; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: none; fill-rule: nonzero; opacity: 1;" vector-effect="non-scaling-stroke"  cx="0" cy="0" r="22.2" />
</g>
		<g transform="matrix(1 0 0 1 99.85 -151.35)"  >
<path style="stroke: rgb(0,0,0); stroke-width: 10; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: none; fill-rule: nonzero; opacity: 1;" vector-effect="non-scaling-stroke"  transform=" translate(-639.85, -143.65)" d="M 550.2 60.5 L 605.3000000000001 227.8 L 648.4000000000001 53.5 L 700.5000000000001 234.8 L 729.5000000000001 52.5" stroke-linecap="round" />
</g>
		<g transform="matrix(1 0 0 1 123.9 160.05)"  >
<path style="stroke: rgb(0,0,0); stroke-width: 10; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: none; fill-rule: nonzero; opacity: 1;" vector-effect="non-scaling-stroke"  transform=" translate(-663.9, -455.05)" d="M 616.3 553.2 L 660.4 356.90000000000003 L 711.5 547.2" stroke-linecap="round" />
</g>
		<g transform="matrix(1 0 0 1 -40.5 8.35)"  >
<path style="stroke: rgb(0,0,0); stroke-width: 10; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: none; fill-rule: nonzero; opacity: 1;" vector-effect="non-scaling-stroke"  transform=" translate(-499.51, -303.35)" d="M 554.2 55.5 C 439.6828770483323 94.70491785693636 368.4829100088484 208.98808800183826 383.76759721617924 329.0612963801879 C 399.05228442351006 449.13450475853756 496.61295428011056 541.9353343988137 617.3000000000001 551.2" stroke-linecap="round" />
</g>
		<g transform="matrix(1 0 0 1 264.25 5.35)"  >
<path style="stroke: rgb(0,0,0); stroke-width: 10; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: none; fill-rule: nonzero; opacity: 1;" vector-effect="non-scaling-stroke"  transform=" translate(-804.25, -300.35)" d="M 730.5 57.5 C 833.9803527483232 96.13678750824292 900.9909970186408 196.81036285549175 896.6960027947375 307.1848969586586 C 892.4010085708342 417.5594310618255 817.7698369989472 512.7213112898287 711.6000000000001 543.2" stroke-linecap="round" />
</g>
</g>
</g>
</svg>