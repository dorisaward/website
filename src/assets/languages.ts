import { I18n } from 'aws-amplify/utils'

export const setLanguage = I18n.setLanguage

export const getText = (key: Dictionary) => I18n.get(key)

export enum languages {
    en = 'ENGLISH',
    th = 'THAI',
  }

type Dictionary = 'sorry'

type Dictionaries = Record<languages, Record<Dictionary, string>>

export const dictionary: Dictionaries = {
    [languages.en]: {
        'sorry': 'Sorry'
    },
    [languages.th]: {
        'sorry': 'ขอโทษ'
    },
}

export const setUpLanguages = () => {
    I18n.putVocabularies(dictionary)
    setLanguage(languages.en)
}
