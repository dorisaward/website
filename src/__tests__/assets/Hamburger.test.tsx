import { Hamburger } from '../../assets/Hamburger'
import { render } from '@testing-library/react'

const testHeight = 25
const testStrokeWidth = 10

describe('Hamburger', () => {
    it('renders correctly given all props', () => {
        // Given
        const { baseElement } = render(
            <Hamburger width={testHeight} height={testHeight} strokeWidth={testStrokeWidth} stroke={'red'}/>
        )

        // Then
        expect(baseElement).toMatchSnapshot()
    })

    it('renders correctly given minimum props', () => {
        // Given
        const { baseElement } = render(
            <Hamburger height={testHeight} strokeWidth={testStrokeWidth} width={undefined}/>
        )

        // Then
        expect(baseElement).toMatchSnapshot()
    })
})