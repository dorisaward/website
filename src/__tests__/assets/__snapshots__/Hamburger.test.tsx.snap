// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hamburger renders correctly given all props 1`] = `
<body>
  <div>
    <svg
      height="25"
      viewBox="0 0 25 25"
      width="25"
    >
      <line
        stroke="red"
        stroke-linecap="round"
        stroke-width="10"
        x1="10"
        x2="15"
        y1="10"
        y2="10"
      />
      <line
        stroke="red"
        stroke-linecap="round"
        stroke-width="10"
        x1="10"
        x2="15"
        y1="12.5"
        y2="12.5"
      />
      <line
        stroke="red"
        stroke-linecap="round"
        stroke-width="10"
        x1="10"
        x2="15"
        y1="15"
        y2="15"
      />
    </svg>
  </div>
</body>
`;

exports[`Hamburger renders correctly given minimum props 1`] = `
<body>
  <div>
    <svg
      height="25"
      viewBox="0 0 25 25"
      width="25"
    >
      <line
        stroke="black"
        stroke-linecap="round"
        stroke-width="10"
        x1="10"
        x2="15"
        y1="10"
        y2="10"
      />
      <line
        stroke="black"
        stroke-linecap="round"
        stroke-width="10"
        x1="10"
        x2="15"
        y1="12.5"
        y2="12.5"
      />
      <line
        stroke="black"
        stroke-linecap="round"
        stroke-width="10"
        x1="10"
        x2="15"
        y1="15"
        y2="15"
      />
    </svg>
  </div>
</body>
`;
